.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+**What is the purpose of this Pull Request?**
+
+_Adjust some error handling on shell scripts_
+
+**What was done to achieve this purpose?**
+
+_Add the exit command in the end of azion-framework-manager call._
+
+**How to test if this really works?**
+
+_Doing the deploy with azioncli and receive the expected errors_
+
+**Who can help review this Merge Request?**
+
+_Who are the best people to review this Merge Request?_
+
+**Is closing some issue?**
+
+_No._
```

Comment:
Actually we should probably keep this with comments, and fill when opening a PR right?
```suggestion
<!--- Fix issue xyz -->

**What was done to achieve this purpose?**
<!-- Describe the key points of implementation  -->

**How to test if this really works?**
<!-- Manual and automated tests that can be executed  -->

**Who can help review this Merge Request?**
<!-- Suggest with @ people inside the Contributors section  -->

**Is closing some issue?**
<!-- Yes/No  -->
```

Reply:
sorry, i misunderstood the usage of PR template. :)